src/redis-benchmark.c

```diff
@@ -308,6 +327,71 @@ static redisContext *getRedisContext(const char *ip, int port,
     return NULL;
 }
 
+/* Wrapper around redisSecureConnection to avoid hiredis_ssl dependencies if
+ * not building with TLS support.
+ */
+static int benchmarkSecureConnection(redisContext *c, const char **err) {
+#ifdef USE_OPENSSL
```

Comment:
@yossigo @filipecosta90 
i see that both `cliSecureConnection` and `writeConn` are cloned here with no changes.
maybe that justifies creating a `cli-common.c` file?

Reply:
Yes, that makes sense.

Reply:
@oranagra @yossigo added the `cli_common.h`and `cli_common.c` files. I added the header file to define:
```
typedef struct cliSSLconfig {
    /* Requested SNI, or NULL */
    char *sni;
    /* CA Certificate file, or NULL */
    char *cacert;
    /* Directory where trusted CA certificates are stored, or NULL */
    char *cacertdir;
    /* Client certificate to authenticate with, or NULL */
    char *cert;
    /* Private key file to authenticate with, or NULL */
    char *key;
} cliSSLconfig;
``` 
Do you guys agree with it? It's a common SSL config required both by redis-cli and redis-benchmark.

Reply:
yeah, seems good